commons/serialization/src/main/scala/sbt/pickling/CustomPicklerUnpickler.scala

```diff
@@ -214,13 +208,21 @@ trait LowPriorityCustomPicklerUnpickler {
   private implicit def staticOnly = scala.pickling.static.StaticOnly
 
   // FIXME this could theoretically work for M<:Map[String,A] and use a CanBuildFrom for M?
-  implicit def stringMapPickler[A](implicit valuePickler: SPickler[A], valueUnpickler: Unpickler[A], valueTag: FastTypeTag[A], mapTag: FastTypeTag[Map[String, A]]): SPickler[Map[String, A]] with Unpickler[Map[String, A]] = new SPickler[Map[String, A]] with Unpickler[Map[String, A]] {
+  implicit def stringMapPickler[A](implicit valuePickler: SPickler[A], valueUnpickler: Unpickler[A], valueTag: FastTypeTag[A],
+    mapTag: FastTypeTag[Map[String, A]],
+    keysPickler: SPickler[List[String]], keysUnpickler: Unpickler[List[String]]): SPickler[Map[String, A]] with Unpickler[Map[String, A]] = new SPickler[Map[String, A]] with Unpickler[Map[String, A]] {
```

Comment:
since it's an implementation detail, we want to embed the keysPickler here I think, the way we do in some of the other picklers (i.e. put a `val keysPickler = implicit[SPickler[List[String]]` in the body of the class below).